### DISCUSSION TITLE
Drawer flashes sometimes

### DISCUSSION THREAD

**Discussion by pszafer** [saadeghi/daisyui#1430] [Q&A]
Hi, I use drawer withouth drawer-mobile so it should be visible only on mobile.
But sometimes when navigating between pages it is shown for a second and dissapear quickly.
Is there something I can do to fix this behavior?
I can't reproduce it consistently, most of the time it just works.
I'm using Next JS 13.

**Comment by saadeghi** [comment]
I think it's either because one of these:
- the CSS gets applied after the page loads
- the value for the checkbox that controls the drawer might be a dynamic and it's `checked` by default until page loads.

So, 
- make sure your CSS file is getting loaded early
- make sure the checkbox that controls the drawer is not checked by default

If these didn't help, I would need a sample repo so I can reproduce the issue and see what's wrong.
